.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -238,29 +238,49 @@ jobs:
           ref: ${{ needs.workflow-context.outputs.sha }}
           path: ${{ env.NEXTSTRAIN_GITHUB_DIR }}
 
+      - name: Verify nextstrain-pathogen.yaml file
+        run: >
+          if [[ ! -f nextstrain-pathogen.yaml ]]; then
+            echo "To use this workflow, there must be a 'nextstrain-pathogen.yaml' file present in the repository root";
+            exit 1;
+          fi
+
       - name: Set up Nextstrain runtime ${{ matrix.runtime }}
         uses: ./.git/nextstrain/.github/actions/setup-nextstrain-cli
         with:
           cli-version: ">=8.3.0"
           runtime: ${{ matrix.runtime }}
 
       - name: Run ingest
+        id: ingest
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: ingest
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
 
       - name: Run phylogenetic
+        id: phylogenetic
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: phylogenetic
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
 
       - name: Run nextclade
+        id: nextclade
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: nextclade
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
+
+      - name: Verify a workflow ran
+        run: |
+          # shellcheck disable=SC2242
+
+          # if we see at least one success, we're good
+          echo "INGEST ATTEMPTED=${{ steps.ingest.outputs.run-attempted }}"
+          echo "PHYLOGENETIC ATTEMPTED=${{ steps.phylogenetic.outputs.run-attempted }}"
+          echo "NEXTCLADE ATTEMPTED=${{ steps.nextclade.outputs.run-attempted }}"
```

Comment:
Minor suggestion to output these to the `$GITHUB_STEP_SUMMARY` so that it's easy to scan the workflow attempts from the GH Action summary instead of having to dig into the job logs.

Reply:
pushed an update — was that what you were thinking of?

Reply:
Yup, that's what I was thinking, except small typo `$"GITHUB_STEP_SUMMARY"` -> `"$GITHUB_STEP_SUMMARY"`

Reply:
>_< thanks -- pushed a fix, gonna merge this once CI finishes

Reply:
![Screenshot 2024-06-13 at 10 18 08 AM](https://github.com/nextstrain/.github/assets/40774762/e095fb83-961f-410c-81a2-0f9903d8e2a0)

🎉